### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by velyan** [rust-skia/rust-skia#124]
I am getting the following error when trying to build the latest master on MacOS Mojave 10.14.5

```
rust-skia/skia-bindings/skia/include/private/SkNx_sse.h:525:24: error: useof undeclared identifier '__builtin_ia32_psrldqi128_byteshift', err: true
thread 'main' panicked at 'Unable to generate bindings: ()', src/libcore/result.rs:999:5
```

I was able to build an older commit `ac8b54d7b9fad479b1a473978b346c5c1a6a35f7 `
I tried `cargo clean`, but no luck
Thanks

**Comment by pragmatrix** [comment]
@velyan Thank you for reporting this!

I can not reproduce the problem, but have an idea what's causing it. 

An attempt to fix the underlying problem is here #125. If it doesn't, please provide the full build output of skia-bindings with `cargo build -vv` (a `touch skia-bindings/src/bindings.cpp` should trigger that without a full rebuild of skia).

**Comment by velyan** [comment]
Thanks @pragmatrix!

Eventually, I was able to build. 
It worked after installing LLVM 7.0.1 via brew install.
